pkgs/by-name/ve/veridian/package.nix

```diff
@@ -0,0 +1,155 @@
+{
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  cmake,
+  python3,
+  ninja,
+  pkg-config,
+  verilator,
+  verible,
+  fmt,
+  boost182,
+  sv-lang,
+  catch2_3,
+  openssl,
+  lib,
+  writeText,
+}:
+
+let
+  fmtver = "11.1.3";
+  fmtsrc = fetchFromGitHub {
+    owner = "fmtlib";
+    repo = "fmt";
+    tag = fmtver;
+    hash = "sha256-6r9D/csVSgS+T/H0J8cSR+YszxnH/h2V2odi2s6VYN8=";
+  };
+
+  fmtlib = fmt.overrideAttrs (oldAttrs: {
+    version = fmtver;
+    src = fmtsrc;
+  });
+
+  boostlib = boost182;
```

Comment:
We should ideally use newer boost versions.
I understand that creating compatibility patches can be difficult, so I won't complain if you don't.